R/format_p.R

```diff
@@ -46,17 +46,7 @@ format_p <- function(p, stars = FALSE, stars_only = FALSE, name = "p", missing =
   }
 
   if (digits == "apa") {
-    text <- ifelse(is.na(p), NA,
-      ifelse(p < 0.001, "< .001***",
-        ifelse(p < 0.01, "< .01**",
-          ifelse(p < 0.05, "< .05*",
-            ifelse(p > .999, "> .999",
-              paste0("= ", format_value(p, 3))
-            )
-          )
-        )
-      )
-    )
+    digits <- 3
   } else if (is.character(digits) && grepl("^scientific", digits)) {
     digits <- tryCatch(
```

Comment:
the `if` statement previously returned `text`, the formatted p-value. I think the revision you're suggesting work when you don't put the first if-statement into an if-else-chain, but just start in line 50 with 

`if (is.character(digits) && grepl("^scientific", digits)) {`

Reply:
and simply `if (digits == "apa") digits <- 3`.

Reply:
ah thanks, yeah, misread something there